tests/robot-tests/tests/admin/bau/prerelease_and_amend.robot

```diff
@@ -0,0 +1,302 @@
+    ...    /prerelease-access#preReleaseAccess-publicList
+    set suite variable    ${RELEASE_URL}
+    user changes to analyst1
+    user navigates to admin frontend    ${RELEASE_URL}/prerelease/content
+    user waits until h1 is visible    Forbidden
+
+Approve amendment for a scheduled release and check warning text
+    user changes to bau1
+    ${day}=    get current datetime    %-d    2
+    ${month}=    get current datetime    %-m    2
+    ${month_word}=    get current datetime    %B    2
+    ${year}=    get current datetime    %Y    2
+    user navigates to admin frontend    ${RELEASE_URL}/status
+    user clicks button    Edit release status
+    user clicks radio    Approved for publication
+    user enters text into element    id:releaseStatusForm-internalReleaseNote    Approved by prerelease UI tests
+    user clicks radio    Immediately
+    user checks page contains
+    ...    Pre-release users will not have access to a preview of the release if it is published immediately.
+    user waits until page contains element    xpath://label[text()="On a specific date"]/../input
+    user clicks radio    On a specific date
+    user checks page contains
+    ...    Pre-release users will have access to a preview of the release 24 hours before the scheduled publish date.
+    user waits until page contains    Publish date
+    user enters text into element    id:releaseStatusForm-publishScheduled-day    ${day}
+    user enters text into element    id:releaseStatusForm-publishScheduled-month    ${month}
+    user enters text into element    id:releaseStatusForm-publishScheduled-year    ${year}
+    user clicks button    Update status
+    user waits until h2 is visible    Confirm publish date
+    user clicks button    Confirm
+
+    user checks summary list contains    Current status    Approved
+    user checks summary list contains    Scheduled release    ${day} ${month_word} ${year}
+    user waits for release process status to be    Scheduled    %{WAIT_MEDIUM}
+
+Validate prerelease has not started for Analyst user
+    user changes to analyst1
+    user navigates to admin frontend    ${RELEASE_URL}/prerelease/content
+
+    user waits until h1 is visible    Pre-release access is not yet available
+    user checks breadcrumb count should be    2
+    user checks nth breadcrumb contains    1    Home
+    user checks nth breadcrumb contains    2    Pre-release access
+
+    ${day}=    get current datetime    %d    1
+    ${month}=    get current datetime    %m    1
+    ${year}=    get current datetime    %Y    1
+    ${time_start}=    format uk to local datetime    ${year}-${month}-${day}T00:00:00    %-d %B %Y at %H:%M
+    ${time_end}=    format uk to local datetime    ${year}-${month}-${day}T23:59:00    %-d %B %Y at %H:%M
+    user checks page contains    Pre-release access will be available from ${time_start} until ${time_end}.
+
+Start prerelease
+    user changes to bau1
+    ${day}=    get current datetime    %-d    1
```

Comment:
I think (not as part of this PR) that what you were suggesting earlier would be really good in terms of looking for things to pull into common keywords.

One candidate for me would be this pattern:

```
    ${day}=    get current datetime    %-d    1
    ${month}=    get current datetime    %-m    1
    ${month_word}=    get current datetime    %B    1
    ${year}=    get current datetime    %Y    1
    .....
    .....
    user enters text into element    id:releaseStatusForm-publishScheduled-day    ${day}
    user enters text into element    id:releaseStatusForm-publishScheduled-month    ${month}
    user enters text into element    id:releaseStatusForm-publishScheduled-year    ${year}
```

which could be boiled down to a single line:

```
    user enters scheduled published date to be X number of days from today   1 
```

It could also be applied to existing keywords like `user approves release for scheduled publication` to make them simpler too.

Reply:
Noted for next time